src/main/kotlin/org/rust/ide/annotator/fixes/AddStructFieldsFix.kt

```diff
@@ -105,4 +158,68 @@ class AddStructFieldsFix(
 
         return null
     }
+
+    private fun defaultValueExprFor(factory: RsPsiFactory, fieldDecl: RsFieldDecl, ty: Ty): RsExpr {
+        return when (ty) {
+            is TyBool -> factory.createExpression("false")
+            is TyInteger -> factory.createExpression("0")
+            is TyFloat -> factory.createExpression("0.0")
+            is TyStr -> factory.createExpression("\"\"")
+            is TyReference -> {
+                defaultValueExprFor(factory, fieldDecl, ty.referenced)
+            }
+            is TyStruct -> {
+                val structName = ty.item.name ?: return factory.createExpression("()")
+                when (structName) {
+                    "String" -> factory.createExpression("String::from(\"\")")
+                    "Vec" -> factory.createExpression("Vec::new()")
+                    "HashMap" -> factory.createExpression("HashMap::new()")
+                    else -> {
+                        if (recursive) {
+                            val structLiteral = factory.createStructLiteral(structName)
+                            fillStruct(
+                                factory,
+                                structLiteral.structLiteralBody,
+                                ty.item.namedFields,
+                                ty.item.namedFields,
+                                false,
+                                recursive)
+                            structLiteral
+                        } else {
+                            factory.createExpression("()")
+                        }
+                    }
+                }
+            }
+            is TyEnum -> {
+                when (ty.item.name) {
+                    "Option" -> factory.createExpression("None")
+                    else -> factory.createExpression("()")
+                }
+            }
+            is TyArray -> {
+                factory.createExpression("[]")
+            }
+            is TyTuple -> {
+                factory.createExpression(ty.types.map { defaultValueExprFor(factory, fieldDecl, it).text }.joinToString(prefix = "(", separator = ",", postfix = ")"))
+            }
+            is TyUnknown -> {
+                val typeName = fieldDecl.typeReference?.text ?: return factory.createExpression("()")
```

Comment:
Hmm, this path seems not much useful? I'll venture to assume that this path was added just because current tests are not std-aware. To make them so, just add `override fun getProjectDescriptor() = WithStdlibRustProjectDescriptor` line to `AddStructFieldsFixTest ` class.